pkgs/development/python-modules/torchinfo/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchPypi
+, python
+, buildPythonPackage
+, pythonOlder
+, pytorch
+, pytestCheckHook
+, torchvision
+}:
+
+buildPythonPackage rec {
+  pname = "torchinfo";
+  version = "1.6.2";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-vf3TADqIX4RtCU3dN/lBK4aRg3wud/KkK9u5XGnBbO4=";
+  };
+
+  propagatedBuildInputs = [
+    pytorch
+    torchvision
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  disabledTests = [
+    # Skip as it downloads pretrained weights (require network access)
+    "test_eval_order_doesnt_matter"
+  ];
+
+  pythonImportsCheck = [ "torchvision" ];
```

Comment:
Sanity check, should I replace this with `"torchinfo"`? Came up in another PR

Reply:
@SuperSandro2000 I had to edit `torchinfo` in https://github.com/NixOS/nixpkgs/pull/222273/files#diff-3528d9b3fd4c65314034db8dd9d529ff644b1c2037e28ecfadae217825729746R46 anyway, so I updated `pythonImportsCheck` in a separate commit